src/shaders/hillshade.fragment.glsl

```diff
@@ -14,9 +14,10 @@ void main() {
 
     vec2 deriv = ((pixel.rg * 2.0) - 1.0);
 
-    // we multiply the slope by a scale factor based on the cosin of the pixel's approximate latitude
+    // We multiply the length by a scale factor based on the cosin of the pixel's approximate latitude
     float scaleFactor = cos(radians((u_latrange[0] - u_latrange[1]) * (1.0 - v_pos.y) + u_latrange[1]));
-    float slope = atan(1.25 * length(deriv)) / scaleFactor;
+    // We also multiply the length by an arbitrary z-factor of 1.25
+    float slope = atan(1.25 * length(deriv) / scaleFactor);
```

Comment:
Great catch! I think this is correct based on https://gis.stackexchange.com/questions/14750/using-srtm-global-dem-for-slope-calculation/40464#40464 which AJ referenced in #4807 
>Compute the slope (as rise:run or a percent) using a Mercator projection, then divide the result by the cosine of the latitude.

and before we were dividing the slope as an _angle_ not a percent 😳